app/models/__init__.py

```diff
@@ -758,6 +758,7 @@
             DELAY_KEY,
             EMAIL_FORMAT_KEY,
             GIT_BRANCH_KEY,
+            IMAGE_TYPE_KEY,
```

Comment:
Do you really need to add this key here at the API level if the value is already stored in the meta-data?